data.sql

```diff
@@ -16,3 +16,47 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Devimon', '2017-05-12', 11.0, true, 5);
 
+---- TASK 2-------
+
+-- Inserting the first animal (Charmander) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Charmander', '2020-02-08', -11.0, false, 0, 'Fire-type');
+
+-- Inserting the second animal (Plantmon) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Plantmon', '2021-11-15', -5.7, true, 2, 'Grass-type');
+
+-- Inserting the third animal (Squirtle) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Squirtle', '1993-04-02', -12.13, false, 3, 'Water-type');
+
+-- Inserting the fourth animal (Angemon) with a negative weight
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Angemon', '2005-06-12', -45.0, true, 1, 'Holy-type');
+
+-- Inserting the fifth animal (Boarmon)
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Boarmon', '2005-06-07', 20.4, true, 7, 'Ground-type');
+
+-- Inserting the sixth animal (Blossom)
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Blossom', '1998-10-13', 17.0, true, 3, 'Grass-type');
+
+-- Inserting the seventh animal (Ditto)
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES ('Ditto', '2022-05-14', 22.0, true, 4, 'Normal-type');
+
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 101448.png
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 101524.png
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 101556.png
+
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 103209.png
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 102931.png
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 103024.png
+
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 103520.png
+
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 104118.png
+
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 105353.png
+-- c:\Users\Danskids\Downloads\Screenshot 2023-09-30 105441.png
```

Comment:
- [ ] Please remove these unnecessary comments from your working directory files, because they look unprofessional and also can put a person into confusion.